compiler-cli/src/hex.rs

```diff
@@ -109,13 +109,33 @@ with a new one?";
     let new_key = auth.create_and_store_api_key()?;
 
     if let Some(previous) = previous {
+        if previous.username != new_key.username {
+            if let Some(previous_username) = previous.username {
+                println!(
+                    "
+Your previous Hex API key was created with username `{}` which is different from the username
+used to create the new Hex API key. You have to delete the key `{}` manually at https://hex.pm",
```

Comment:
We wrap before 80 columns! Could you call the `wrap` function on this please.

Reply:
done!

Reply:
There's some extra newlines in this string! The wrap function will preserve them, so please remove the first one and escape the others with a `\`

Reply:
double uppps, should be corrected now 🙏🏻